app/models/ApprovalProcess.scala

```diff
@@ -18,4 +18,4 @@ package models
 
 import play.api.libs.json.JsObject
 
-case class ApprovalProcess(id: String, process: JsObject)
+case class ApprovalProcess(id: String, meta: ApprovalProcessMeta, process: JsObject)
```

Comment:
as discussed ApprovalProcessMeta format

Reply:
This makes the retrieval of the summary list easier in the repository as the code doesn't have to navigate json within the meta info - it can just access as class attributes of the ApprovalProcess.  If this can be done as easily accessing json then can change, just not sure advantage of having everything as JsObjects as code needs to know structure of the meta data.